### PR TITLE
Extract strings

### LINKED ISSUE #43: Extract hardcoded strings to strings file in EditFragment

### PR DESCRIPTION
**Pull request by frcroth**
Closes #43

### CODE REVIEW

**Review by PassiDel — APPROVED** [review]

**Comment by frcroth** [comment]
Waiting on a review from @henrijaeger before merging

**Review by henrijaeger — APPROVED** [review]
Looks great! Thank you!

**Pull request merged**
